internal/pkg/template/templates/addons/ddb/env/cf.yml

```diff
@@ -0,0 +1,45 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The name of the environment being deployed.
+
+Resources:
+  {{logicalIDSafe .Name}}:
+    Metadata:
+      'aws:copilot:description': 'An Amazon DynamoDB table for {{.Name}}'
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: !Sub ${App}-${Env}-{{.Name}}
+      AttributeDefinitions:{{range .Attributes}}
+        - AttributeName: {{.Name}}
+          AttributeType: "{{.DataType}}"{{end}}
+      BillingMode: PAY_PER_REQUEST
+      KeySchema:
+        - AttributeName: {{.PartitionKey}}
+          KeyType: HASH{{ if .SortKey }}
+        - AttributeName: {{.SortKey}}
+          KeyType: RANGE{{end}}{{if .HasLSI}}
+      LocalSecondaryIndexes:{{range .LSIs}}
+        - IndexName: {{.Name}}
+          KeySchema:
+            - AttributeName: {{.PartitionKey}}
+              KeyType: HASH
+            - AttributeName: {{.SortKey}}
+              KeyType: RANGE
+          Projection:
+            ProjectionType: ALL{{end}}{{end}}
+
+Outputs:
+  {{envVarName .Name}}:
+    Description: "The name of this DynamoDB."
+    Value: !Ref {{logicalIDSafe .Name}}
```

Comment:
Noticed this is an existing one for wkld addons but I am not sure why we want to output this...how do we expect users to use it 🤔

Reply:
In their application code, people need the name of the table. For example, with [boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb.html#DynamoDB.Table), you may write
```python
user_table = dynamo_client.Table(os.environ.get('USER_TABLE_NAME'))
```

With env-level ddb, you'd 
1. Export the ddb table name
2. Write in service manifest:
```
variables:
   USER_TABLE_NAME:
      from_cfn: UserDDBTableName
```

Maybe I can add the commented `Export` block for the DDB name output as well.

Reply:
Maybe we should export `TableName: !Sub ${App}-${Env}-{{.Name}}` which is the real table name? Or export "AWS::DynamoDB::Table" which is https://github.com/aws/copilot-cli/pull/4289/files#diff-f3cbab28817cc024d734b1c7fddb2144cf94e7c5af19eb69549833492e309d64R39-R44

Reply:
I'm not sure I follow! 

The current implementation uses `!Ref LogicalID` which returns the real table name: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-table.html#aws-resource-dynamodb-table-return-values. Can you clarify the difference between this and your suggestion?

Reply:
:homer-disappearing: